.teamcity/Gradle_Util/buildTypes/WarmupEc2Agent.kt

```diff
@@ -0,0 +1,24 @@
+package Gradle_Util.buildTypes
+
+import common.gradleWrapper
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildType
+import jetbrains.buildServer.configs.kotlin.v2019_2.Requirement
+import jetbrains.buildServer.configs.kotlin.v2019_2.RequirementType
+
+object WarmupEc2Agent : BuildType({
+    uuid = "980bce31-2a3e-4563-9717-7c03e184f4a4"
+    name = "Warmup EC2 Agent"
+    id("Gradle_Util_WarmupEc2Agent")
```

Comment:
I suppose you also want to attach a VCS root to the build, right?

Reply:
Yes, but isn't that already set by default? How does it then work on the _Util/Performance_ builds as there I do not see any specific VCS root attached.

Reply:
Thanks for the hint. Done https://github.com/gradle/gradle/commit/63f6128a1d1d6a3f5b2b1ceb22066a1847748ee0